tests/adapter/dbt/tests/adapter/dbt_clone/test_dbt_clone.py

```diff
@@ -0,0 +1,203 @@
+import os
```

Comment:
Would love some ideas on ways to slim down or better these tests (not sure if we need all of the same setup that the defer_state tests had) also would love to expand coverage of tests in ways that are easy ex. testing against opposite of default value for `can_clone_table` or that we have a solid expectation of what should happen if a user adds a custom macro to change the default value.